vistrails/gui/uvcdat/definedVariableWidget.py

```diff
@@ -261,7 +261,7 @@ def addVariable(self, var, type_='CDMS'):
         ## print "Added variable"
         ## self.emit(QtCore.SIGNAL('setupDefinedVariableAxes'), var)
 
-    def deleteVariable(self, varid):
+    def deleteVariable(self, varid,count):
```

Comment:
I am keeping a count on how many times the method is being called. Depending on the first or the second call the method either returns the var or throws the error.